app/pages/sections/funding-sources/contextCreator.js

```diff
@@ -1,11 +1,27 @@
 import manifest from './manifest.json';
 import { baseUrl } from '../../../config';
 
-export const getContext = ({ orderId }) => {
+const generateFundingSourceOptions = ({ fundingSource }) => {
+  const fundingMap = manifest.questions[0].options.map(option => ({
+    ...option,
+    checked: option.value === fundingSource ? true : undefined,
+  }));
+  return fundingMap;
+};
+
+const generateQuestionsContext = ({ fundingSource }) => (
+  manifest.questions.map(question => ({
+    ...question,
+    options: generateFundingSourceOptions({ fundingSource }),
```

Comment:
pass in the `question` to `generateFundingSourceOptions` then in the `generateFundingSourceOptions` function you should have to do `manifest.questions[0]`